### PR TITLE
Post-History should include dates of postings to python-ideas

### PR DESCRIPTION
**Pull request by phdru**
Refs: https://mail.python.org/pipermail/python-dev/2017-October/149990.html

### CODE REVIEW

**Comment by warsaw** [comment]
We had a discussion on python-dev about this (buried in the PEP 517 thread), and Guido expressed the opinion of dropping python-list and adding python-ideas.  I agree with that, although it's a process change and will require notification to the relevant mailing lists.

Can you please make that change to this PR?

File: pep-0001.txt [hunk 1]
```diff
@@ -379,7 +379,8 @@ optional and are described below.  All other headers are required. ::
   * Requires: <pep numbers>
     Created: <date created on, in dd-mmm-yyyy format>
   * Python-Version: <version number>
-    Post-History: <dates of postings to python-list and python-dev>
+    Post-History: <dates of postings to python-list, python-dev
+    and/or python-ideas>
```
Comment (warsaw) [inline pep-0001.txt]: This should now say `dates of postings to python-ideas and/or python-dev`.

File: pep-0001.txt [hunk 2]
```diff
@@ -434,8 +435,9 @@ Content-Type header is present.
 
 The Created header records the date that the PEP was assigned a
 number, while Post-History is used to record the dates of when new
-versions of the PEP are posted to python-list and/or python-dev.  Both
-headers should be in dd-mmm-yyyy format, e.g. 14-Aug-2001.
+versions of the PEP are posted to python-list, python-dev and/or
+python-ideas.  Both headers should be in dd-mmm-yyyy format, e.g.
+14-Aug-2001.
```
Comment (warsaw) [inline pep-0001.txt]: This should also now say `posted to python-ideas and/or python-dev`.

**Comment by phdru** [comment]
@warsaw Done.

File: pep-0001.txt [hunk 3]
```diff
@@ -379,7 +379,7 @@ optional and are described below.  All other headers are required. ::
   * Requires: <pep numbers>
     Created: <date created on, in dd-mmm-yyyy format>
   * Python-Version: <version number>
-    Post-History: <dates of postings to python-list and python-dev>
+    Post-History: <dates of postings to python-dev and/or python-ideas>
```
Comment (warsaw) [inline pep-0001.txt]: Actually, this needs to be `python-ideas and/or python-dev`.  The reasoning is that python-dev must always be included but python-ideas is optional.

Comment (warsaw) [reply pep-0001.txt]: Same comment below.

**Comment by phdru** [comment]
@warsaw Done.

**Pull request merged**